_benchmarking/bm_tobyfit_fit/test_bm_tobyfit_fit_largeData.m

```diff
@@ -0,0 +1,356 @@
+classdef test_bm_tobyfit_fit_largeData < TestCase
+%TEST_BM_TOBYFIT_FIT_LARGEDATA largeData Benchmark class for fit()
+% This set of benchmarks uses "large" sized sqw objects (10^9 pixels).
+% The parameters that are varied in this set of benchmarks are:
+%   - nDims: the dimensions of the sqw objects to combine: 1,2 or 3
+%   - dataSet: the number of sqw objects in the array
+%   - nProcs: the number of processors the benchmarks will run on
+    
+    properties
+        function_name;
+        common_data;
+        func_handle = @slow_func;
+        params = {[250 0 2.4 10 5],@demo_FM_spinwaves,10^-1};
+        dataSize = 'large';
+    end
+    
+    methods
+        function obj = test_bm_tobyfit_fit_largeData(test_class_name)
+            %TEST_BM_TOBYFIT_FIT_LARGEDATA Construct an instance of this class
+            if ~exist('test_class_name','var')
+                test_class_name = 'test_bm_tobyfit_fit_largeData';
+            end
+
+            obj = obj@TestCase(test_class_name);
+            pths = horace_paths;
+            obj.common_data = pths.bm_common;
+        end
+        
+%         function test_bm_tobyfit_fit_1D_largeData_smallNumber_1procs(obj)
```

Comment:
noting that its the large data that gets all tests commented but it would still be better to say so explicitly and why